calico/security/encrypt-cluster-pod-traffic.md

```diff
@@ -19,21 +19,27 @@ This how-to guide uses the following {{site.prodname}} features:
 
 ### Before you begin...
 
-**Supported**
-
 The following platforms using only IPv4:
+
 - Kubernetes, on-premises
 - EKS using Calico CNI
 - EKS using AWS CNI
 - AKS using Azure CNI
 
-All platforms listed above will encrypt pod-to-pod traffic. Additionally, when using AKS or EKS, host-to-host traffic will also be encrypted, including host-networked pods.
+**Supported traffic for encryption**
+
+- Pod-to-pod traffic
```

Comment:
Pod-to-pod traffic is misleading and potentially wrong.

For example, we don't encrypt pod -> pod if it's on the same host - we only encrypt the traffic on the host-to-host network hop.

I'd probably say that we support "host-to-host encryption for pod traffic, and optionally for direct node to node communication as well"

Reply:
Updated using your terms, and also updated Warnings to use same.